zetta_utils/bcube/bcube.py

```diff
@@ -46,6 +48,22 @@ def ndim(self) -> int:
         """float of dimensions."""
         return len(self.bounds)
 
+    # NOTE: Cannot use VecT as return type since TypeVars are not callable.
+    @property
+    def start(self) -> Vec3D:  # pragma: no cover
```

Comment:
Motion to kill the ND-ness of the BoundingCube

Reply:
As noted in the comment, I have to be able to construct the Vec3D to return it, and VecT isn't callable. This is probably ugliness that we have to accept until the BoundingCube is changed.